### PR TITLE
docs: Install Docs Update

### PR DESCRIPTION
**Pull request by Rosiak**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: doc/Installation/Installing-LibreNMS.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 [Ubuntu 16.04](http://docs.librenms.org/Installation/Ubuntu-image/)
 
 [CentOS 7](http://docs.librenms.org/Installation/CentOS-image/)
 
-If you want to install yourself then we have some new documentation which should make it easy.
+##Manuel
+If you want to install manually then we have some documentation which should make it easy.
 
-> Please note the following docs are new and may not be 100% complete, please provide feedback on your experience.
+**Please note from January 10th 2018, the minimum supported PHP version will be 5.6.4** [More]
```
Comment (laf) [inline doc/Installation/Installing-LibreNMS.md]: This needs to be in all the other install docs as well as most likely people will skip straight to those. We also don't need to say from Jan the 10th here and just say, min supported version is 5.6.4 now.

File: doc/Installation/Installing-LibreNMS.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
 source: Installation/Installing-LibreNMS.md
 # Installing LibreNMS
 
-We have some pre-built VirtualBox images you can use to get started:
+##Images
+We have some pre-built VirtualBox images you can use to get started. Please note that these images are pre-built for VirtualBox, hence they might not work with other hypervisors:
```
Comment (laf) [inline doc/Installation/Installing-LibreNMS.md]: Possibly just adding `hence they might not work with other hypervisors without converting them:`

File: doc/Installation/Installing-LibreNMS.md (hunk 1 above)
Comment (murrant) [reply doc/Installation/Installing-LibreNMS.md]: Agreed, much clearer if we just say the minimum version is 5.6.4 for newcomers.

File: doc/Installation/Installing-LibreNMS.md [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
 
 [Ubuntu 16.04](http://docs.librenms.org/Installation/Ubuntu-image/)
 
 [CentOS 7](http://docs.librenms.org/Installation/CentOS-image/)
 
-If you want to install yourself then we have some new documentation which should make it easy.
+##Manuel
```
Comment (murrant) [inline doc/Installation/Installing-LibreNMS.md]: Manuel -> Manual

File: doc/Installation/Installing-LibreNMS.md [hunk 4]
```diff
@@ -29,3 +31,5 @@ If the above are missing or incomplete then you can try our old docs below.
 [Debian/Ubuntu Lighttpd](http://docs.librenms.org/Installation/Installation-Ubuntu-1404-Lighttpd/)
 
 [Debian/Ubuntu Nginx](http://docs.librenms.org/Installation/Installation-Ubuntu-1404-Nginx/)
+
+[More]: https://community.librenms.org/t/change-to-php-minimum-version-5-6/2542
```
Comment (murrant) [inline doc/Installation/Installing-LibreNMS.md]: Possibly unnecessary if we just are posting the minimum version as 5.6.4.  We'll advertise this link on social media.

**Pull request merged**